src/bosh-google-cpi/action/create_vm.go

```diff
@@ -184,7 +185,14 @@ func (cv CreateVM) findZone(zoneName string, disks []DiskCID) (string, error) {
 	return "", fmt.Errorf("Could not find zone %q", zoneName)
 }
 
+func isGcpImageURL(s string) bool {
+	return strings.HasPrefix(s, "https://www.googleapis.com/compute/v1/projects/")
```

Comment:
@ljfranklin The problem is that as far as I can tell, even after giving someone permission to my images, they can't access them with the name, only the url.

I had considered changing the CPI code to always accept urls instead of names but that seemed like a huge change to your code base.